conftest.py

```
@@ -1115,123 +1121,137 @@ def prepared_plan(
-                # Wait for guest info to become available (VMware only)
-                if source_provider.type == Provider.ProviderType.VSPHERE:
-                    source_provider.wait_for_vmware_guest_info(
-                        provider_vm_api, timeout=class_plan_config.get("guest_agent_timeout", 120)
-                    )
-            elif source_vm_power == "off":
-                source_provider.stop_vm(provider_vm_api)
+                conflicting.append("migrate_shared_disks")
+            if conflicting:
+                raise ValueError(
+                    f"skip_clone=True is incompatible with {conflicting}; these options require the cloning phase."
+                )
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)